src/figures/figure2.jl

```diff
@@ -0,0 +1,65 @@
+""" Figure 2: drug combination. """
+
+function helper(g1_Bliss_model1, g2_Bliss_model1, i, title, subPlabel, label1, label2, meanGS2)
+    t = LinRange(0.0, 95.0, 189)
+    p = plot(t, g1_Bliss_model1, label=label1, xlabel="time [hr]", lw=2, ylabel="cell number", color="darkseagreen3", title=title, titlefont = Plots.font("Helvetica", 12), guidefont=Plots.font("Helvetica", 12), xtickfont=Plots.font("Helvetica", 12),ytickfont=Plots.font("Helvetica", 12), bottom_margin=1.5cm, fg_legend = :transparent, top_margin=1.25cm, left_margin=1.25cm, right_margin=1.25cm)
+    plot!(t, g2_Bliss_model1, label=label2, titlefont = Plots.font("Helvetica", 12), lw=2, color="mediumpurple2", guidefont=Plots.font("Helvetica", 12), xtickfont=Plots.font("Helvetica", 12),ytickfont=Plots.font("Helvetica", 12), bottom_margin=1.5cm, fg_legend = :transparent, top_margin=1.25cm, left_margin=1.25cm, right_margin=1.25cm)
+    plot!(t, meanGS2[1, 1:189, i], label="G1 exp", titlefont = Plots.font("Helvetica", 12), lw=2, color="darkgreen", guidefont=Plots.font("Helvetica", 12), xtickfont=Plots.font("Helvetica", 12),ytickfont=Plots.font("Helvetica", 12), bottom_margin=1.5cm, fg_legend = :transparent, top_margin=1.25cm, left_margin=1.25cm, right_margin=1.25cm)
+    plot!(t, meanGS2[2, 1:189, i], label="G2 exp", titlefont = Plots.font("Helvetica", 12), lw=2, color="mediumpurple4", guidefont=Plots.font("Helvetica", 12), xtickfont=Plots.font("Helvetica", 12),ytickfont=Plots.font("Helvetica", 12), bottom_margin=1.5cm, fg_legend = :transparent, top_margin=1.25cm, left_margin=1.25cm, right_margin=1.25cm)
+    annotate!(-25.0, 87.0, text(subPlabel, :black, :left, Plots.font("Helvetica Bold", 15)))
```

Comment:
The y-position here is way too high. That's why they're on the wrong plot.

Reply:
Thanks! Fixed.

Reply:
![image](https://user-images.githubusercontent.com/34812750/111674783-9f961c00-87d9-11eb-8789-b4a7918b9e75.png)